load_data.js

```diff
@@ -0,0 +1,52 @@
+(function(window){
+  window.extractData = function() {
+    var ret = $.Deferred();
+
+    FHIR.oauth2.ready(function(smart){
+      var patient = smart.patient;
+      var pt = patient.read();
+     
+      $.when(pt).done(function(patient){
+		var gender = patient.gender;
+		var dob = new Date(patient.birthDate);		 
+		var day = dob.getDate();
+		var monthIndex = dob.getMonth();

```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth
This is a 0 based index.  You'd need to add 1.